server/index.js

```diff
@@ -31,6 +31,38 @@ app.get('/api/products', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.get('/api/products/:id', (req, res, next) => {
+  const sql = `
+  select "name",
+         "imageUrl",
+         "price",
+         "description",
+         "profileName",
+         "profileDescription",
+         "flex",
+         "shapeName",
+         "shapeDescription",
+         "edgeTechName",
+         "edgeTechDescription",
+         "abilityLevel",
+         "terrain",
+         "size"
+    from "snowboards"
+    join "shapes" using ("shapeId")
+    join "edgeTech" using ("edgeTechId")
+    join "profileTypes" using ("profileId")
+    join "sizes" using ("productId")
+   where "productId" = $1;
+  `;
+  const params = [Number(req.params.id)];
+  db.query(sql, params)
+    .then(result => {
+      const products = result.rows;
+      res.status(200).json(products);
+    })
```

Comment:
We should throw a 404 if we can't find the product. 

also, the client should receive an object instead of an array